lib/vmdb/settings/walker.rb

```diff
@@ -1,8 +1,8 @@
 module Vmdb
-  class Settings
-    module Walker
-      PASSWORD_FIELDS = %i(bind_pwd password amazon_secret ssh_key_data ssh_key_unlock become_password vault_password security_token).to_set.freeze
+  module SettingsWalker
```

Comment:
So, I slightly alluded to this being on purpose in the PR description:

> * Changes the `Vmdb::Settings::Walker` to be a nested module only (no class definitions). While the file path hasn't changed, the module is now `Vmdb::SettingsWalker` _**(slightly breaks normal conventions)**_

But left out the reasons why because most people complain that I write too much... but since you asked for it, here you go (sucker)!

So, basically I left out changing this for 3 reasons:

1. Makes a cleaner diff for people to grok the actual changes.  File renames usually make the github diffs blobs of red and green instead of the few lines that were actually changed
2. The file is currently `lib/vmdb/settings/walker.rb` in it's current file structure, and if I just changed the name to match it, it would turn into `lib/vmdb/settings/settings_walker.rb`, which seemed redundant.
3. I honestly don't think it should live under `lib/vmdb/settings/` anymore, but because I expected this would probably be a debate anyway, I decided to get a first pass review from people to understand what the changes actually were, and re-commit when a proper name has been decided on.

That all said, open to suggestions on where this should live now, and really have no preference.  I was thinking like `lib/vmdb/concerns/settings_walker.rb`, but not really tied to it.  Doesn't have to move either, but figured since it is used out of the context of `Vmdb::Settings` in a few places now, it probably should.